static/app/views/ddm/layout.tsx

```diff
@@ -111,6 +112,29 @@ export const DDMLayout = memo(() => {
   );
 });
 
+const SplitDivider = styled((props: DividerProps) => (
```

Comment:
we could probably get a little more out of the DefaultSplitDivider, or a base divider. sucks to have to copy+paste the cursor and user-select stuff each time.